proto/internal/temporal/server/api/adminservice/v1/request_response.proto

```diff
@@ -288,32 +287,6 @@ message DescribeClusterResponse {
   string http_address = 14;
 }
 
-message ListClustersRequest {
-  int32 page_size = 1;
-  bytes next_page_token = 2;
-}
-
-message ListClustersResponse {
-  repeated temporal.server.api.persistence.v1.ClusterMetadata clusters = 1;
```

Comment:
operator service seem to return a different type here?

Reply:
I think this is ok. But we should try to migrate use cases to operator service before merge this.